dbt/adapters/dremio/credentials.py

```diff
@@ -53,6 +53,14 @@ class DremioCredentials(Credentials):
     def type(self):
         return "dremio"
 
+    @property
+    def unique_field(self):
+        """
+        Hashed and included in anonymous telemetry to track adapter adoption.
+        Pick a field that can uniquely identify one team/organization building with this adapter
+        """
+        return self.software_host if self.cloud_host is None else self.cloud_host
```

Comment:
dbt itself suggests the use of host. Using DC project ID here would lead to a lack of symmetry, as software would still use host and DC would use project ID.